src/python/pants/backend/docker/target_types.py

```diff
@@ -227,5 +227,17 @@ class DockerImageTarget(Target):
         "build context, after being packaged if applicable.\n\n"
         "By default, will use a Dockerfile from the same directory as the BUILD file this target "
         "is defined in. Point at another file with the `source` field, or use the `instructions` "
-        "field to have the Dockerfile contents verbatim directly in the BUILD file."
+        "field to have the Dockerfile contents verbatim directly in the BUILD file.\n\n"
+        "Dependencies on upstream (or base) images defined by another `docker_image` are infered "
```

Comment:
The (or base) makes it sound like those are two separate things. Really they're different names for the same concept, right? If so, maybe `upstream/base` would be clearer.

Reply:
Thanks